### PR TITLE
Electrochemical reactions

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
```SUMMARY: Content "adding few electrochemical recipes/variations"```

#### Purpose of change
Electrochemistry is a thing, quite a big thing actually. Lets reflect that by giving players access to some common industry processes that are using electrochemistry - and a way to utilize that electrolysis kit that is sitting abandoned on the shelf.

Introducing the [Chloralkali process](https://en.wikipedia.org/wiki/Chloralkali_process) (electrolysis of brine).

#### Describe the solution
Added new recipe variants for hydrochloric (muriatic) acid and lye, changed recipes for bleach and oxidizer powder to reflect their requirement for electricity (they were completely bogus without).

#### Testing
Looked over the recipes in game, checked requirements and batch times, seems to be working as intended.

### CODE REVIEW

**Comment by Soup-de-Loop** [comment]
Looks like a great way to provide additional utility for a tool that doesn't see much use, also good to see platinum's catalytic abilities being used in more places.

File: data/json/recipes/recipe_medsandchemicals.json [hunk 1]
```diff
@@ -820,6 +827,28 @@
...
+  {
+    "type": "recipe",
+    "result": "chem_muriatic_acid",
+    "id_suffix": "by_electrolysis",
+    "byproducts": [ [ "lye", 2 ] ],
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "skill_used": "cooking",
+    "skills_required": [ "electronics", 2 ],
+    "difficulty": 5,
```
Comment (I-am-Erk) [inline data/json/recipes/recipe_medsandchemicals.json]: I think igniting chlorine and hydrogen successfully with an electric arc welder in a chemical process probably requires a bit more than cooking 5.

**Comment by I-am-Erk** [comment]
In general I think your difficulties are a bit low, for the cooking skill. When there's a chemistry skill (which there will be once we have enough recipes to justify it, so thanks for contributing to that) it might be another matter, but carrying out a chloralkali reaction able to produce usable goods is more demanding than making pizza, so we usually put it into the higher echelons of the skill.

**Comment by 1n17** [comment]
It would perhaps be worthwhile to remember that while cooking is overloaded (and, if I do say so myself - artificially stratified) chemistry_textbook has max level of 6. Another thing would be that chemistry (most crafting skills really, but chemistry especially) is not something you get better with practice (past a certain point), you just get faster - you either have the intelligence required to solve encountered problems/improvise or you don't and are stuck with following recipes word for word (and if you can't do that - well, again, no amount of training will help you).

Alas, I see no problem with bumping cooking requirement if that is the general sentiment.

**Comment by I-am-Erk** [comment]
Cooking is definitely overloaded, that's why we'll eventually move this stuff to a chemistry skill. In the meantime I'd say chemistry stuff should probably run around skill ranges 4-6.

File: data/json/recipes/recipe_medsandchemicals.json (hunk 1 above)
Comment (1n17) [reply data/json/recipes/recipe_medsandchemicals.json]: Indeed, I suppose one also needs to know how welder works and how to use it.

**Pull request merged**